internal/resources/ibm/ibm_cos_bucket.go

```diff
@@ -404,18 +482,251 @@ func (r *IbmCosBucket) MonthlyDataRetrievalCostComponent() *schema.CostComponent
 		UnitMultiplier:  decimal.NewFromInt(1),
 		MonthlyQuantity: q,
 		ProductFilter: &schema.ProductFilter{
-			VendorName:       strPtr("ibm"),
-			Region:           strPtr(r.Location),
-			Service:          strPtr(("cloud-object-storage")),
-			ProductFamily:    strPtr("iaas"),
-			AttributeFilters: []*schema.AttributeFilter{},
+			VendorName:    strPtr("ibm"),
+			Region:        strPtr(r.Location),
+			Service:       strPtr(("cloud-object-storage")),
+			ProductFamily: strPtr("iaas"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{
+					Key: "planName", Value: strPtr(r.Plan),
+				},
+			},
 		},
 		PriceFilter: &schema.PriceFilter{
 			Unit: strPtr(u),
 		},
 	}
 }
 
+func (r *IbmCosBucket) OneRateMonthlyAverageCapacityCostComponent() *schema.CostComponent {
+	var q *decimal.Decimal
+
+	if r.MonthlyAverageCapacity != nil {
+		q = decimalPtr(decimal.NewFromFloat((*r.MonthlyAverageCapacity)))
+	}
+
+	return &schema.CostComponent{
+		Name:            "Storage Capacity",
+		Unit:            "GB",
+		UnitMultiplier:  decimal.NewFromInt(1),
+		MonthlyQuantity: q,
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("ibm"),
+			Region:        strPtr(oneRateMapping[r.Location]),
+			Service:       strPtr(("cloud-object-storage")),
+			ProductFamily: strPtr("iaas"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{
+					Key: "planName", Value: strPtr(r.Plan),
```

Comment:
I'm not sure how this is finding a price. I only see 1 sku in the pricing db with a planName of 'cos-one-rate-plan' and it has a region of global?